### PR TITLE
Documentation for WiLight device integration - Initial

### PR DESCRIPTION
**Pull request by leofig-rj**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1553
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/wilight.markdown [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+
+There is currently support for the following device types within Home Assistant:
+
+- [Cover](#cover)
+- [Fan](#fan)
+- [Light](#light)
+- [Switch](#switch)
+
+## Cover
```
Comment (klaasnicolaas) [inline source/_integrations/wilight.markdown:25]: I don't think that this adds value to the docs 🤔

File: source/_integrations/wilight.markdown [hunk 2]
```diff
@@ -0,0 +1,124 @@
+---
+title: WiLight
+description: Instructions on how to integrate WiLight devices into Home Assistant.
+logo: wilight.png
```
Comment (klaasnicolaas) [inline source/_integrations/wilight.markdown:4]: Please add a logo to the [brand repository](https://github.com/home-assistant/brands)

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Is there already a pull request to the core repository?

File: source/_integrations/wilight.markdown (hunk 2 above)
Comment (leofig-rj) [reply source/_integrations/wilight.markdown:4]: Done!
Please, can you check if it is OK?
Thank you!

File: source/_integrations/wilight.markdown (hunk 1 above)
Comment (leofig-rj) [reply source/_integrations/wilight.markdown:25]: I have no problem removing it.
You can do it?
Or do I have to do a new PR?
Thanks

File: source/_integrations/wilight.markdown (hunk 2 above)
Comment (leofig-rj) [reply source/_integrations/wilight.markdown:4]: PR in brand repository #1553

Comment (frenck) [reply source/_integrations/wilight.markdown:4]: It needs to be removed from this PR...

**Comment by frenck** [comment]
@leofig-rj I can't find the PR in the code base, could you please point us to it?

**Pull request closed without merging**